tests/show_techsupport/test_techsupport_no_secret.py

```diff
@@ -0,0 +1,99 @@
+import os
+import pprint
+import pytest
+import time
+import logging
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import skip_release
+from log_messages import *
```

Comment:
>* [](http://example.com/codeflow?start=25&length=1)

Please try not use wildcard in import statement. Just be specific.

Reply:
Fixed, remove this line from test_techsupport_no_secret.py, also fix import from log_messages in test_techsupport.py